### ISSUE TITLE
Incorrect syntax highlighting with template variables

### ISSUE THREAD

**Issue by klemenoslaj** [angular/vscode-ng-language-service#641]
**Describe the bug**

After the template variable, the template highlighting is broken unless it is suffixed with `;`. Highlighting is correct with this extension disabled.

**To Reproduce**

```html
<p *ngIf="true; let value">
  Paragraph
</p>
```

Correct with `;`

```html
<p *ngIf="true; let value;">
  Paragraph
</p>
```

**Screenshots**

<img width="207" alt="Screenshot 2020-02-17 at 10 16 24" src="https://user-images.githubusercontent.com/7548247/74639734-b4375900-516e-11ea-8dd9-69ecd0550ee6.png">

<img width="202" alt="Screenshot 2020-02-17 at 10 16 30" src="https://user-images.githubusercontent.com/7548247/74639732-b3062c00-516e-11ea-9e28-5f48ab48665a.png">

**Comment by CarlHenrik** [comment]
I've had this issue as well for about a month. Not sure which version introduced it.
Thanks for finding the semicolon workaround, this was driving me nuts.

**Comment by ayazhafiz** [comment]
This is a known issue at #575. There is a PR to fix it at #609. Sorry for the wait. Marking as duplicate.
